person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, parent_permission: true, name: 'Unknown')
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] In your Person class you should have a  method for adding rentals it may look something like this :point_down: 

```ruby
def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```